spec/extractors/web_of_science_spec.rb

```diff
@@ -44,5 +44,24 @@
         # rubocop:enable Lint/PercentStringArray
       end
     end
+
+    context 'when there is only one result' do
+      before do
+        stub_request(:get, 'https://api.clarivate.com/api/wos?count=2&databaseId=WOS&firstRecord=1&usrQuery=AU=,%20AND%20OG=Stanford%20University')
+          .to_return(status: 200, body: '{"Data":{"Records": { "records": {"REC": "one"}}},'\
+            '"QueryResult": {"RecordsFound": 1}}')
+
+        allow(extractor.send(:client)).to receive(:page_size).and_return(2)
+      end
+      it 'calls the block on single result' do
+        results = []
+        extractor.each do |records|
+          results << records
+        end
+        # rubocop:disable Lint/PercentStringArray
+        expect(results).to eq %w["one"]
```

Comment:
May I suggest `expect(results).to eq ['"one"']`?  I think this is more readable.

Reply:
Agree.